src/blackduck-security-action/utility.ts

```diff
@@ -15,6 +15,71 @@ import {debug, warning, info} from '@actions/core'
 import * as https from 'https'
 import {HttpClient} from 'typed-rest-client/HttpClient'
 import {getSSLConfig, getSSLConfigHash, createHTTPSAgent} from './ssl-utils'
+import semver from 'semver'
+
+/**
+ * Compares two version strings using semantic versioning.
+ * Falls back to string comparison if either version is invalid.
+ * @param version1 First version to compare
+ * @param version2 Second version to compare
+ * @returns true if version1 < version2, false otherwise
+ */
+export function isVersionLessThan(version1: string, version2: string): boolean {
```

Comment:
can we make use of "{lt} from 'semver' " library instead of writing custom logic?